service/src/main/kotlin/no/nav/su/se/bakover/service/AccessCheckProxy.kt

```diff
@@ -760,10 +765,40 @@ open class AccessCheckProxy(
             kontrollsamtale = object : KontrollsamtaleService {
                 override fun kallInn(
                     sakId: UUID,
-                    saksbehandler: NavIdentBruker
+                    kontrollsamtale: Kontrollsamtale
                 ): Either<KunneIkkeKalleInnTilKontrollsamtale, Unit> {
                     assertHarTilgangTilSak(sakId)
-                    return services.kontrollsamtale.kallInn(sakId, saksbehandler)
+                    return services.kontrollsamtale.kallInn(sakId, kontrollsamtale)
+                }
+
+                override fun nyDato(
+                    sakId: UUID,
+                    dato: LocalDate
+                ): Either<KunneIkkeSetteNyDatoForKontrollsamtale, Unit> {
+                    assertHarTilgangTilSak(sakId)
+                    return services.kontrollsamtale.nyDato(sakId, dato)
+                }
+
+                override fun hentNestePlanlagteKontrollsamtale(sakId: UUID): Either<KunneIkkeHenteKontrollsamtale, Kontrollsamtale> {
+                    assertHarTilgangTilSak(sakId)
+                    return services.kontrollsamtale.hentNestePlanlagteKontrollsamtale(sakId)
+                }
+
+                override fun hentPlanlagteKontrollsamtaler(clock: Clock): Either<KunneIkkeHenteKontrollsamtale, List<Kontrollsamtale>> {
+                    return services.kontrollsamtale.hentPlanlagteKontrollsamtaler(clock)
+                }
+
+                override fun opprettPlanlagtKontrollsamtale(vedtak: Vedtak): Either<KunneIkkeKalleInnTilKontrollsamtale, Kontrollsamtale> {
+                    assertHarTilgangTilSak(vedtak.behandling.sakId)
+                    return services.kontrollsamtale.opprettPlanlagtKontrollsamtale(vedtak)
+                }
+
+                override fun oppdaterNestePlanlagteKontrollsamtaleStatus(
```

Comment:
Skal alle disse 5 nye funksjonene kunne kalles fra Routes? Hvis ikke har vi en slik kastException... funksjon

Reply:
Bare to av de ja. Fikser